apps/console/src/__tests__/__mocks__/next-image.tsx

```
@@ -0,0 +1,8 @@
+/**
+ * Stub for next/image — renders a plain <img> element.
+ */
+function NextImage(props: React.ImgHTMLAttributes<HTMLImageElement>) {
+  return <img {...props} />;
```

Biome (2.4.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

(lint/a11y/useAltText)